assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -87,7 +99,9 @@ export default function WidgetAreaRenderer( { slug, totalAreas, contextID } ) {
 		select( CORE_WIDGETS ).getWidgetArea( slug )
 	);
 	const widgets = useSelect( ( select ) =>
-		select( CORE_WIDGETS ).getWidgets( slug )
+		select( CORE_WIDGETS ).getWidgets( slug, {
+			modules: viewableModules ? viewableModules : undefined,
```

Comment:
The module filtering logic in `getWidgets` requires a truthy value, so why not use `viewableModules` here without the additional condition?
```suggestion
			modules: viewableModules,
```

Reply:
Oh, yeah, it's possible to do that but I found it made the code a lot less immediately-understandable… I know it's just a bit above here where `viewableModules` is declared, but I found this was a bit more explicit/readable.

We can remove it if you feel strongly, but I thought this was a bit more clear about what we're doing.